rules/expect-expect.js

```diff
@@ -33,16 +34,18 @@ module.exports = {
         : ['expect'];
 
     return {
-      'CallExpression[callee.name=/^it|test$/]'(node) {
-        unchecked.push(node);
-      },
-      [`CallExpression[callee.name=/^${assertFunctionNames.join('|')}$/]`]() {
-        // Return early in case of nested `it` statements.
-        for (const ancestor of context.getAncestors()) {
-          const index = unchecked.indexOf(ancestor);
-          if (index !== -1) {
-            unchecked.splice(index, 1);
-            break;
+      CallExpression(node) {
```

Comment:
Using the selector query syntax became a little confusing now having to support MemberExpressions, so all CallExpression logic lives in this visitor function now. 😃